ds-caselaw-ingester/lambda_function.py

```diff
@@ -116,7 +116,10 @@ def extract_lambda_versions(versions: List[Dict[str, str]]) -> List[Tuple[str, s
 
 
 def store_metadata(uri: str, metadata: dict) -> None:
-    tdr_metadata = metadata["parameters"]["TDR"]
+    try:
+        tdr_metadata = metadata["parameters"]["TDR"]
+    except KeyError:
+        return  # TODO: Get non-TDR metadata added to the metadata file
```

Comment:
Would a printed error message make sense here?

Reply:
I'm not going to commit this until we have something else for the metadata, the TODO is to remind me to add it!